modules/rostests/apitests/kernel32/GetComputerNameEx.c

```diff
@@ -328,6 +328,7 @@ TestReturnValues()
     CHAR ComputerName[128];
     DWORD ComputerNameSize = 0;
     INT ErrorCode;
+    BOOL ret;
```

Comment:
Can you name this `Ret` please to be consistent with the rest of the file?

Reply:
Done.